packages/tabs/src/styled/StyledTab.ts

```diff
@@ -5,55 +5,120 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import PropTypes from 'prop-types';
-import styled from 'styled-components';
-import classNames from 'classnames';
-import TabStyles from '@zendeskgarden/css-tabs';
-import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
+import styled, { DefaultTheme, css, ThemeProps } from 'styled-components';
+import { retrieveComponentStyles, getColor, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import stripUnit from 'polished/lib/helpers/stripUnit';
 
 const COMPONENT_ID = 'tabs.tab';
 
 interface IStyledTabProps {
-  isHovered?: boolean;
-  isFocused?: boolean;
-  isActive?: boolean;
   disabled?: boolean;
   isSelected?: boolean;
 }
 
+/**
+ * A high specificity is needed to apply the border-color in vertical orientations
+ */
+const colorStyles = ({ theme, isSelected }: IStyledTabProps & ThemeProps<DefaultTheme>) => {
+  const selectedColor = getColor('primaryHue', 600, theme);
+
+  return css`
+    color: ${isSelected ? selectedColor : 'inherit'};
+
+    &&& {
+      border-color: ${isSelected && 'currentColor'};
+    }
```

Comment:
Ok so when we run into situations like this, where all other options have been exhausted, it's time to use the dreaded `!important`. Adding `&` until we "win" is brittle.

One other option would be to add the following line under the `${StyledTab}` portion of the `StyledTabs` vertical styling:

```css
&[aria-selected='true'] {
  border-color: currentColor;
}
```

The latter is probably what I would opt for, but YMMV.

Reply:
I will use the `aria-selected` selector. But, from our offline conversation, this selector is a side-effect of the `useSelection()` hook. I'm still unsure if this is something that we should bake into the styles.

Reply:
JK, that doesn't work with horizontal rules and adds additional specificity issues. Going to use `!important`